src/test/java/org/alfresco/repo/tagging/TaggingServiceImplTest.java

```diff
@@ -2426,5 +2435,89 @@ public Void execute() throws Throwable
             nodeRefPropInterceptor.setFilterOnGet(true);
         }
     }
+    
+    /**
+     * ALF-21875
+     */
+    @Test
+    public void testTagScopeALF_21875() {

```

Comment:
The Brace should start from new line, see the [coding standarts](https://community.alfresco.com/docs/DOC-4658-coding-standards)